analyticsdataserver/settings/local.py

```diff
@@ -74,4 +76,11 @@
 # hosts.
 ALLOWED_HOSTS = ['localhost', '127.0.0.1', '::1', 'analyticsapi']
 
+JWT_AUTH.update({
```

Comment:
out of curiosity, how does this get updated for other environments like prod?

Reply:
This gets set in the configuration repo: https://github.com/edx/configuration/blob/f3d6a97a405321cc54f86727755a2d5f33efb5a1/playbooks/roles/edxapp/defaults/main.yml#L1104